src/lib/components/events/event-item.svelte

```diff
@@ -58,12 +60,12 @@
         <time>
           {info.month}
           {info.day} at {info.time}
-          {#if isRecurring}(recurring){/if}
+          {#if isRecurring}<br />Recurring{/if}
```

Comment:
![image](https://user-images.githubusercontent.com/8981287/165003506-a72342ca-ac24-4861-a91b-4cbe51ff6075.png)

Is there a way to verify that this works? The only recurring event doesn't actually indicate recurring

Reply:
that's weird bc i remember it did indicate recurring. will check